granary/microformats2.py

```diff
@@ -173,12 +186,15 @@ def object_to_json(obj, ctx={}, trim_nulls=True):
       objs = obj.get('object', [])
       if not isinstance(objs, list):
         objs = [objs]
-      ret['properties'][prop] = ret['properties'][prop + '-of'] = \
-          [o.get('url') for o in objs]
+      ret['properties'][prop + '-of'] = ret['properties'][prop] = [
+        o['url'] if 'url' in o and len(o) == 1
```

Comment:
Sometimes (e.g. Twitter faves), the object is a dict with just `{ "url": "..." }`. I didn't like the result of running that through `object_to_json` ... it gave something like
```
"like-of": [{ 
  "type": "h-cite", 
  "properties": {
    "name": ["http://twitter.com/..."], 
    "url": ["http://twitter.com/..."]
  }
}]
```
and converted to HTML poorly